CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## v2.3.1
+  Update how the passwords are sent to kdb5_util when creating realm db
+  Changed guard for realm db creation from test -e to File.exist?
+
```

Comment:
Put this under Unreleased, please

Reply:
apologies, wasn't sure if I needed to add that or not.